docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+# Installing packages
+This documentation dives deeper into how a user can configure packages for installation inside the EIB image. Furthermore, it explains the **RPM resolution** process that EIB goes through so it can ensure that configured packages will be successfully installed even in an **air-gapped** environment.
+
+Below you can find a table of contents for easier navigation:
+
+1. [Supported systems](#supported-systems)
+2. [Configure packages for installation](#configure-packages-for-installation)
```

Comment:
Does it work to use the built-in syntax of prefixing all items in a numbered list as `1.`? I'm not sure if the embedded unordered list messes that up and is why you manually number them. If we can, it's better to use the markdown syntax where possible so we can take advantage of any accessibility features it provides.